src/engine/renderer/tr_init.cpp

```diff
@@ -1015,6 +1015,27 @@ ScreenshotCmd screenshotPNGRegistration("screenshotPNG", ssFormat_t::SSF_PNG, "p
 			Log::Notice("%sMissing GPU vertex skinning, models are not hardware-accelerated.", Color::ToString( Color::Red ) );
 		}
 
+		switch ( glConfig2.textureRGBA16BlendAvailable )
+		{
+			case 1:
+				if ( r_highPrecisionRendering.Get() )
+				{
+					Log::Notice( "%sUsing GL_RGBA16 with GL_FRAMEBUFFER_BLEND.", Color::ToString( Color::Green ) );
+				}
+				else
+				{
+					Log::Notice( "%sHaving GL_RGBA16 with GL_FRAMEBUFFER_BLEND.", Color::ToString( Color::Green ) );
+				}
+				break;
+			case -1:
+				Log::Notice( "%sAssuming GL_RGBA16 with GL_FRAMEBUFFER_BLEND.", Color::ToString( Color::Yellow ) );
```

Comment:
This will say "assuming GL_RGBA16" even if the feature is disabled, so nothing will be done with this assumption. Maybe it's better to have a message like 'Explicitly disabling RGBA16' in all cases where the cvar is disabled? Or alternatively, don't make any of the messages depend on the cvar.

Reply:
This GfxInfo command prints a report about what's available to the engine, it is always right that the engine assumes the feature is available or not, even if unused.

The rest of the code already says “Using that extension”, even if related extension is only used in a GLSL code that is disabled. I actually went too verbose with the Using/Having distinction, but I still want to report the extension is assumed.